src/utils/index.js

```diff
@@ -956,71 +996,15 @@ export const unlockTool = (currentToolLevels, toolType) => {
 }
 
 /**
- * @param {Farmhand.state} state
- * @return {Object}
+ * @param {farmhand.state} state
+ * @return {farmhand.state}
  */
 export const transformStateDataForImport = state => {
   const sanitizedState = { ...state }
-  const { id } = sanitizedState
 
   const rejectedKeys = ['version']
   rejectedKeys.forEach(rejectedKey => delete sanitizedState[rejectedKey])
 
-  // Update old data models
-
-  if (sanitizedState.field) {
-    // Update plot data
-    sanitizedState.field = sanitizedState.field.map(row =>
-      row.map(plot => {
-        if (plot === null) {
-          return null
-        }
-
-        const { isFertilized, ...rest } = plot
-
-        return {
-          ...rest,
-
-          // Convert from isFertilized (boolean) to fertilizerType (enum)
-          fertilizerType:
-            rest.fertilizerType ||
-            (isFertilized ? fertilizerType.STANDARD : fertilizerType.NONE),
-        }
-      })
-    )
-  }
-
-  const { tools: unlockedTools } = getLevelEntitlements(
-    levelAchieved(farmProductsSold(sanitizedState.itemsSold))
-  )
-
-  for (const tool of Object.keys(unlockedTools)) {
-    sanitizedState.toolLevels = unlockTool(sanitizedState.toolLevels, tool)
-  }
-
-  if (
-    !sanitizedState.showHomeScreen &&
-    sanitizedState.stageFocus === stageFocusType.HOME
-  ) {
-    sanitizedState.stageFocus = stageFocusType.SHOP
-  }
-
-  // TODO: Remove these cowInventory and cowForSale transformations after
-  // 3/15/2023
-  sanitizedState.cowInventory = sanitizedState.cowInventory.map(cow => ({
-    ownerId: id,
-    originalOwnerId: id,
-    timesTraded: 0,
-    ...cow,
-  }))
-
-  sanitizedState.cowForSale = {
-    ownerId: '',
-    originalOwnerId: '',
-    timesTraded: 0,
-    ...sanitizedState.cowForSale,
-  }
-
```

Comment:
This function was long overdue for a cleanup, so I removed support for deprecated save files as part of this PR.